_posts/2020-01-20-hacks.md

```diff
@@ -0,0 +1,101 @@
+* Depression affects about 20% of adolescents by the time they become adults. Develop Interfaces or chatbots that can help counsel people with issues like depression or anxiety issues and suicidal tendencies.
+* Personal Health Assistant : The AI in your phone will figure out exactly when and how to wake you up given your health, sleep patterns and time constraints. It will also look into your eating habits and suggest you restaurants and dishes for keeping you in track.
+
+### Algorithms
+
+* Can you devise new algorithms to improve the control of vehicular traffic in crowded cities?
+* Bitcoin – last year it was claimed that the computing power required to keep the network running consumes as much energy as was used by 159 of the world’s nations. Come up with an implementation which solves the energy problem with your own algorithm.
+
+### Fin-Tech
+
+* Digital Banking:  Enable after-hours banking services with an interactive interface using chatbots and video capabilities. Leveraging on technology to build personal teller service for banking transactions or professional banking consultations for customers after business hours as well.
+* Alternative Credit Rating: Access to finance & banking facilities remains limited for the unbanked population & businesses in remote areas. A financial/banking app could potentially bridge this gap by sourcing information that can contribute to an alternative credit rating, and subsequently allow other institutions to leverage on it.
+* Cashback @ Merchants: Finding an ATM can be a challenge and sometimes they may not be in service. Design a solution that allows the withdrawal and deposit of cash from merchants for easier access to cash.
+* Invoice financing: Invoice duplication and fraud cases in invoice financing can be mitigated by creating a database leveraging on a unique ID for each invoice/document.
+* Learning Interface for All: Understanding banking instruments and financial systems remains a challenge in Tier 2-3 and rural areas due to language and smartphone usage barriers. Develop a digital solution for awareness generation for this segment to be able to understand and access relevant financial instruments.
+* Financial and Wealth Management: Wealth Management is resorted to only by the Upper-Class Indians who have significant savings to invest in Alternative Funds. Financial Planning and investment of savings by the middle class and youth is very low. Develop a solution to increase middle class and younger generation participation in Financial Markets to leverage investment of savings.
+
+### Future Mobility
+
+* Study of flow dynamics of people within a city to improve public transport infrastructure
+* Monitoring the driver to keep them alert at all times.
+* Detecting unavoidable Frontal, Rear end, T-bone, and other types of collisions to activate preventive measures.
+* mycardoeswhat.org datasets as a source of inspiration
```

Comment:
New point needed for the dataset ?

Reply:
The website has many datasets related to autonomous driving and vehicular safety. So I added in a new line.